test/functional/tool_wallet.py

```diff
@@ -211,6 +211,56 @@ def test_salvage(self):
 
         self.assert_tool_output('', '-wallet=salvage', 'salvage')
 
+    def test_dump_createfromdump(self):
+        self.start_node(0, ['-wallet=todump', '-wallet=todump2'])
+        self.stop_node(0)
+
+        self.log.info('Checking dump arguments')
+        self.assert_raises_tool_error('No dump file provided. To use dump, -dumpfile=<filename> must be provided.', '-wallet=todump', 'dump');
+
+        self.log.info('Checking basic dump')
+        wallet_dump = os.path.join(self.nodes[0].datadir, "wallet.dump")
+        self.assert_tool_output('', '-wallet=todump', '-dumpfile={}'.format(wallet_dump), 'dump')
+
+        with open(wallet_dump, 'r') as f:
```

Comment:
This is getting caught in lint-python-utf8-encoding for not specifying `encoding="utf8"`.
```suggestion
        with open(wallet_dump, 'r', encoding="utf8") as f:
```

Same with the other `with open`s below (lines 244, 250, 255, and 260).

Reply:
Fixed